### DISCUSSION TITLE
Welcome to bhsa Discussions!

### DISCUSSION THREAD

**Discussion by dirkroorda** [ETCBC/bhsa#10] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to reflect on the particulars of the linguistic annotations to the Hebrew Bible.

Have a look around in the [README](https://github.com/ETCBC/bhsa/blob/master/README.md) of this repo and in the [feature documentation](https://etcbc.github.io/bhsa/), for example the [otype](https://etcbc.github.io/bhsa/features/otype/) page where you see the textual objects that are defined in the BHSA.

**Comment by MichoelR** [comment]
Hi! This is a wonderful resource. Tell me, please:  The "word" otype here means a single syntactic atom. Is there a way to use BigTables as listed, but also to have information on _whole_ Hebrew words?
For instance, the first word is B Reishit ("in the beginning") stored as two rows. Same with Ha Shamayim ("the heavens") later in the verse, stored as two rows. Is there a way to know that they are linked in one word?
Thank you!

**Reply by MartijnNaaijer** [reply]
Hi. The feature "trailer" indicates what there is at the end of a word, before the next word. So, B has the value "" (empty string), Reishit has " " (space).

**Reply by MartijnNaaijer** [reply]
For the feature docs, see https://etcbc.github.io/bhsa.

**Reply by MichoelR** [reply]
OMG. I couldn't even _see_ the space when I looked at the file in Excel. So any empty string means there's more to the word. Then there is a weird list of other possibilities, 0 (end of verse), 5 (?), & (seems to be a maqaf), P (petucha), S (setumah), 00_|_S 
(whatever). 
In any case, this takes care of the problem: keep adding to the word as long as trailer is empty. Thank you!

**Reply by dirkroorda** [reply]
You are getting it with the other trailers. This might also help: https://annotation.github.io/text-fabric/tf/writing/hebrew.html
Scroll down to punctuation.

**Comment by Dderekjones** [comment]
Hi,  I noticed the gender for some gentilic name types (nmtp) like Dan and Naphtali have 'u' (unknown) gender.  Trying to determine a method or feature to get the phrase context gender (similar to the phrase dependent part of speech).  Looking at Genesis 46:24 Sentence 45 (2021) 26353, the nmtp is 'pers,gens,topo', but in context, it's referring to the person.   Any help is appreciated,

Thanks,
Derek
